app/src/main/java/io/github/muntashirakon/AppManager/profiles/AppsProfileActivity.java

```diff
@@ -290,7 +291,7 @@ public boolean onOptionsItemSelected(@NonNull MenuItem item) {
     @Override
     protected void onDestroy() {
         if (viewPager != null) {
-            viewPager.removeOnPageChangeListener(this);
+            viewPager.registerOnPageChangeCallback(pageChangeCallback);
```

Comment:
A typo, I think. It should be `unregisterOnPageChangeCallback` and not `registerOnPageChangeCallback`.

Reply:
Ah, nice catch 🤦 , thanks!